include/maxscale/config.hh

```diff
@@ -138,6 +138,7 @@ public:
     std::string   admin_pam_ro_service;         /**< PAM service for read-only users */
     std::string   admin_ssl_key;                /**< Admin SSL key */
     std::string   admin_ssl_cert;               /**< Admin SSL cert */
+    admin_ssl_version_t    admin_ssl_version;   /**< Admin allowed SSL versions */
```

Comment:
Align the variable names at the same level. In addition, add this as the last TLS related variable (after `admin_ssl_ca_cert`) to keep the certificate related parameters grouped together.